patch/core/book.js

```diff
@@ -26,19 +23,20 @@ module.exports = {
 			} else if (item.includes('crystal')) {
 				item = 'crystal';
 			}
-
+			
 			// set clue text based on actor type
 			if (actor === 'Death') {
 				clues.push('Death guards\n' + item);
 			} else if (actor === 'Camilla') {
 				clues.push('Camilla\ndefends\n' + item);
 			} else if (actor === 'merchant') {
-				clues.push(item + '\nfor sale in\n' + location);
```

Comment:
I see the comment here, but I don't fully understand why basic merchants are being excluded. Knowing any given item is for sale in a town, particularly if it's a holy-water blocked one, could significantly affect routing and make the book _really_ worth grabbing.

Reply:
Also, comments should have a space between the `//` and the text of the comment

Reply:
in playing i found getting a merchant notice to always be annoying. you have to go out of your way to get a book and i wanted them to have important(-ish) information, i care about the bosses and orbs the most, merchants were just bottom of the list and never seemed to matter. maybe a check so only merchants behind holy water walls would be better? going out of my way to find a merchant in jova has a whip is just bleck... your call of course but i'd rather not have merchants in the list